<!--igvc_run.launch-->
<launch>


<node pkg="usma_triclops" type="bumblebee_node" name="bumblebee" output="screen"/>
<node pkg="usma_triclops" type="whiteline_node" name="whiteline" output="screen"/>
<node pkg="usma_triclops" type="vision3d_node" name="vision3d" output="screen">
	<remap from="/vision3D/points" to="/vision3D/lines"/>
</node>

</launch>

